lib/robots/dor_repo/gis_delivery/reload_geoserver.rb

```diff
@@ -26,7 +26,11 @@ def perform_work
               }
             )
             begin
-              connection.post(path: 'reload', payload: nil)
+              with_retries(max_tries: Settings.connection_error_max_retries,
+                           handler: retries_handler("Reloading #{setting} Geoserver for #{druid} (#{setting[:url]})"),
+                           rescue: Faraday::TimeoutError) do
+                connection.post(path: 'reload', payload: nil)
+              end
```

Comment:
cribbed from https://github.com/sul-dlss/preservation_robots/blob/7c9e43e4d9a65961ed921993ce4fdfee64874baa/lib/robots/sdr_repo/preservation_ingest/update_catalog.rb#L30-L39 (though unlike the pres bots example, i don't think any `rescue`ing was needed in the `with_retries` block)